plugin/core/views.py

```diff
@@ -176,20 +176,68 @@ def text_document_range_formatting(view: sublime.View, region: sublime.Region) -
     })
 
 
-def minihtml(view: sublime.View, content: Union[str, dict]) -> str:
-    """ Content can be a string or MarkupContent. """
+def minihtml(view: sublime.View, content: Union[str, dict, list], prefer_plain_text: bool) -> str:
```

Comment:
What do you think that instead of `prefer_plain_text: bool` we have `marked_string: bool = False`? 

I know that I could have suggested `plain_text: bool = True` to be the default, 
but I think changing it to `marked_string` makes it more explicit.

There is only one call for now where marked_string would be set to True:
```
return minihtml(self.view, self._hover.get('contents') or "", marked_string=True)
```
the rest of the calls we will just be:
```diff
-   detail = minihtml(self.view, item.get('detail') or "", prefer_plain_text=True)
+  detail = minihtml(self.view, item.get('detail') or "")

Reply:
Wfm

Reply:
I agree the `prefer_plain_text` is a bit vague. `marked_string` communicates better what is happening.

Reply:
To be fair I don't like either current or suggested solution too much. Latter also seems to suggest that you are passing MarkedString while it could be MarkupContent.

I'll try to do something with bitwase flags. I think that would be the most explicit and truthful solution.

Reply:
Done now.